consensus/istanbul/core/roundchange.go

```diff
@@ -85,6 +85,10 @@ func (c *core) handleRoundChange(msg *message, src istanbul.Validator) error {
 		return errInvalidMessage
 	}
 
+	if !c.valSet.CheckInSubList(msg.Hash, rc.View, src.Address()) {
+		return errNotFromCommittee
+	}
```

Comment:
I think a test for round change is required. 
It seems validators excluded in committee don't handle round change.

Reply:
As we discussed offline, it will not exclude any committee member with this change. 
Anyway, we will test round change with "Klaytn-deploy".

Reply:
Because it is all about checking senders and only committee members can send a message, it looks okay to me.